browser/src/ConstraintTable/GnomadNonCodingConstraintTableVariant.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { BaseTable, TooltipAnchor, TooltipHint } from '@gnomad/ui'
+
+import Link from '../Link'
+
+import { renderRoundedNumber } from './constraintMetrics'
+
+const Table = styled(BaseTable)`
+  width: 100%;
+  margin-top: 1rem;
+
+  @media (max-width: 600px) {
+    td,
+    th {
+      padding-right: 10px;
+
+      /* Drop sparkline column */
+      &:nth-child(5) {
+        display: none;
+      }
+    }
+  }
+`
+
+const ViewSurroundingRegion = styled.div`
+  margin-top: 1rem;
+`
+
+type NonCodingConstraint = {
+  start: number
+  stop: number
+  possible: number
+  observed: number
+  expected: number
+  oe: number
+  z: number
+}
+
+type Props = {
+  variantId: string
+  chrom: string
+  nonCodingConstraint: NonCodingConstraint
```

Comment:
@rileyhgrant exactly what I was hoping you'd do here

Reply:
For some reason the variant ID is overlapping with your tick slightly when I look at it in my browser (Chrome). It looks fine in firefox (and in your screenshot).

![image](https://user-images.githubusercontent.com/8867295/197024610-4c051be2-82f8-48d9-80a8-a5808d27deb2.png)